extensions/test/kokkos/types.cpp

```diff
@@ -0,0 +1,121 @@
+// SPDX-FileCopyrightText: 2017-2023 The Ginkgo authors
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <cstring>
+#include <Kokkos_Core.hpp>
+#include <sstream>
```

Comment:
minor nit:

Maybe `Kokkos_Core.hpp` should go into its own section

Reply:
I put it now in the same category as omp, cuda, ...